Samples/HelloWorld.NetCore/NuGet.Config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="Orleans-local-vNext" value="..\..\vNext\Binaries\Debug\" />
+    <remove key="Orleans-local" />
```

Comment:
After clear we don't need Orleans-local removal I think.
What about adding the myget feed here to me it F5 runnable without building Orleans?